.github/ISSUE_TEMPLATE/release_new_version.md

```diff
@@ -0,0 +1,70 @@
+Targeted translation completion date: <!-- E.g. two weeks after feature freeze, e.g. 2021-05-04 -->
+Approximate release date: <!-- E.g. one week after targeted translation completion date to have time for late translations, reviews, RC, e.g. 2021-05-11 -->
+Current state: <!-- Planning|Translations (beta)|Code freeze (rc)|Released -->
+
+**Checklist**
+- [ ] Assign this issue to the release shepherd who is in charge of managing this checklist
+- [ ] Create a milestone and add all relevant issues
+- [ ] Ensure that all issues/PR targeted for this release are done by checking the Project board with an appropriate filter (e.g. [`milestone:"Release 3.8.0"`](https://github.com/orgs/jamulussoftware/projects/2?card_filter_query=milestone%3A"release+3.8.0")). Reminder main developers to review entries in *Waiting on team* state. De-tag unfinished Issues/PRs.
+- [ ] Declare a code freeze by updating this Issue and adding a comment. PRs can still be worked on and may get reviewed, but must not be merged unless agreed explicitly.
+- [ ] Check ./Jamulus -h output against the en-Command-Line-Options.md wiki page and update if necessary.
+- [ ] Write the Changelog based on the list of merged PRs ([example for 3.8.0](https://github.com/jamulussoftware/jamulus/pulls?q=is%3Amerged+milestone%3A"Release+3.8.0"))
+- [ ] Start Website translations
+  - [ ] Create new branch `translate3_y_z` based on branch `release`
+  - [ ] Squash-merge `changes` into `translate3_y_z`
+  - [ ] Check if the list of translators in `tools/create-translation-issues.sh` is up-to-date
+  - [ ] Create a translation Issue for each language, e.g. `../jamulus/tools/create-translation-issues.sh 3.8.0 2021-05-15 web 'Note: The term "Central server" has been replaced with "Directory server"'` (Last argument is optional and should be skipped if there is nothing to add)
+- [ ] Start App translations
+  - [ ] Generate `.ts` files in master via `lupdate`
+  - [ ] Check if the list of translators in `tools/create-translation-issues.sh` is up-to-date
+  - [ ] Create a translation Issue for each language, e.g. `./tools/create-translation-issues.sh 3.8.0 2021-05-15 'Note: The term "Central server" has been replaced with "Directory server"'` (Last argument is optional and should be skipped if there is nothing to add)
+- [ ] [Tag a beta release](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Announce the beta release on Github Discussions. Pin the thread.
+- [ ] Get feedback on the stability of the beta release
+- [ ] Finish Website translations
+  - [ ] Wait for all PRs to be merged
+  - [ ] Decide what to do about missing translations. Depending on the missing changes, consider keeping the old state (if it is still valid) or temporary backing out the relevant file(s) if they would become wrong.
+  - [ ] Merge `translate3_y_z` into `changes` via PR
```

Comment:
We didn't do this until now. The process was:

changes > translate (manual) > release (manual) > changes (via automation)

Any reasons for the change?

Reply:
It was just me misunderstanding the current process. :)
Will update the PR in the coming days.

Reply:
Removed this step (and updated the later one).